### PR TITLE
Fix Many Bugs In Quizmaster & New Setting: Can Give Questions About Past Games

### PR DESCRIPTION
**Pull request by Tommy-XL**

### CODE REVIEW

File: Roles/Neutral/Quizmaster.cs [hunk 1]
```diff
@@ -422,7 +451,7 @@ public override void FixUnsetAnswers()
         int positionForRightAnswer = rnd.Next(0, 3);
 
         if (QuizmasterQuestionType == QuizmasterQuestionType.EjectionColorQuestion)
-            Answer = Quizmaster.lastReportedColor;
+            Answer = Quizmaster.lastExiledColor;
         else if (QuizmasterQuestionType == QuizmasterQuestionType.ReportColorQuestion)
             Answer = Quizmaster.lastReportedColor;
         else if (QuizmasterQuestionType == QuizmasterQuestionType.LastMeetingColorQuestion)
```
Comment (Gurge44) [inline Roles/Neutral/Quizmaster.cs:457]: Why not use a `switch` statement here?

Comment (Tommy-XL) [reply Roles/Neutral/Quizmaster.cs]: Done

**Comment by FuroYT** [comment]
damn, it was a good idea to move it to experimental after all

**Comment by Tommy-XL** [comment]
> damn, it was a good idea to move it to experimental after all

But the Quizmaster's role is still experimental
image

**Comment by FuroYT** [comment]
> But the Quizmaster's role is still experimental image

i know

**Comment by Tommy-XL** [comment]
Well, i think this done

**Pull request merged**